umbraco-cloud/troubleshooting/deployments/merge-conflicts-on-flexible-environments.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Merge conflicts can happen on a flexible environment when the same schema has been changed in both the source and target environment.
+---
+
+# Merge Conflicts on Flexible Environments
+
+A merge conflicts happens when one file or item contains changed in more than one instance that's part of a deployment.
+
+This guide explains how to resolve these merge conflicts and how to avoid them.
+
+## How to Resolve a Merge Conflict on a Flexible Environment
+
+The image below shows a merge conflict happening on a pull from the mainline environment to a flexible environment.
+
+![Pulling changes from the mainline environment failed](images/pulling-failed.png)
+
+To start debugging the merge conflict, a log file containing a list of conflicting files is provided. It's a `.txt` file that can be downloaded and viewed.
+
+### Preparations
+
+Before you start resolving the merge conflict, ensure you have the following things ready:
+
+* The log file containing a list of the conflicting files.
+* The clone link for the flexible environment.
+* The clone link for the mainline environment that flexible environment is connected to.
```

Comment:
Conflicts can happen between any environment, that is merging together. 

Would it make sense to call them source and destination environment?